LSP-eslint.sublime-settings

```diff
@@ -4,10 +4,18 @@
 			"languageId": "javascript",
 			"document_selector": "source.js",
 		},
+		{
+			"languageId": "javascriptreact",
+			"document_selector": ["source.jsx", "source.js.react"],
```

Comment:
This is not a valid selector. It must be a string so: `source.jsx, source.js.react`

Reply:
No, sorry: `source.jsx | source.js.react`

Reply:
Whoops, was copy-pasting from `LSP-typescript`, which is `scopes`, not `document_selector` :slightly_smiling_face: